src/System.Diagnostics.Process/tests/ProcessTests.netcoreapp.cs

```diff
@@ -10,6 +10,7 @@ namespace System.Diagnostics.Tests
     public partial class ProcessTests : ProcessTestBase
     {
         [Fact]
+        [ActiveIssue(31908, TargetFrameworkMonikers.Uap)]
```

Comment:
Should this be added to the class instead?

Reply:
As it's a partial class I didn't put it their as it accidentally disables all the tests for uap then.

Reply:
But you're already adding it to ProcessTests.cs (Also weird enough, this file is named .netcoreapp.cs and disabling tests for uap in here?)

Reply:
oh you are right. the netcoreapp file is also used in uap...